example/20_convnd_bwd_weight_xdl/convnd_bwd_weight_xdl.cpp

```diff
@@ -296,15 +296,53 @@ int main(int argc, char* argv[])
                                   OutElementOp{},
                                   split_k);
 
-    if(!conv->IsSupportedArgument(argument.get()))
+    // alloc work space
+    size_t bwd_weight_workspace_size = conv->GetWorkSpaceSize(argument.get());
+    float ave_time                   = 0.f;
+    if(bwd_weight_workspace_size > 0)
```

Comment:
I think the predicate should be "whether the datatype is bf16 or not". ```bwd_weight_workspace_size``` is a result of the predicate, not the predicate itself.

Reply:
I agree. Changed.